subt/main.py

```diff
@@ -582,9 +706,9 @@ def join(self):
 
 def main():
     import argparse
-    from osgar.lib.config import config_load
+    from osgar.lib.config import load as config_load
```

Comment:
I believe this should not be here. In master it is called `config_load`.

Reply:
OK, thanks